php-unsafe-file-upload/Triangle-blog_raw_e.php

```
@@ -0,0 +1,271 @@
+<?php
+// 图片压缩处理函数
+function compressImage($sourcePath, $destPath, $maxSize = 800 * 1024) {
+    // 验证图片有效性
+    $imageInfo = getimagesize($sourcePath);
+    if (!$imageInfo) return false;
+    list($width, $height, $type) = $imageInfo;
+    $mime = image_type_to_mime_type($type);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$mime'. (undefined)

(UnusedLocalVariable)